src/ui/MainMenu.cpp

```diff
@@ -9341,197 +10444,45 @@ namespace MainMenu {
-		host_lan_button->setHighlightColor(makeColor(255, 255, 255, 255));
-		host_lan_button->setColor(makeColor(255, 255, 255, 255));
-		host_lan_button->setText("Host LAN Party");
-		host_lan_button->setFont(smallfont_outline);
-		host_lan_button->setWidgetSearchParent(window->getName());
-		host_lan_button->setWidgetBack("back_button");
-		host_lan_button->setWidgetUp("local");
-		host_lan_button->setWidgetDown("host_online");
-		host_lan_button->setCallback([](Button&){soundActivate(); createLobby(LobbyType::LobbyLAN);});
-
-		(void)window->addImage(
-		    SDL_Rect{270, 170, 126, 50},
-		    0xffffffff,
-		    "*images/ui/Main Menus/Play/NewGameConnectivity/UI_NewGame_Icon_HostLAN_00B_Unselected.png",
-		    "host_lan_image"
-		);
-
-		auto host_online_button = window->addButton("host_online");
-		host_online_button->setSize(SDL_Rect{96, 232, 164, 62});
-		host_online_button->setBackground("*images/ui/Main Menus/Play/NewGameConnectivity/ButtonStandard/Button_Standard_Default_00.png");
-		host_online_button->setHighlightColor(makeColor(255, 255, 255, 255));
-		host_online_button->setColor(makeColor(255, 255, 255, 255));
-		host_online_button->setText("Host Online Party");
-		host_online_button->setFont(smallfont_outline);
-		host_online_button->setWidgetSearchParent(window->getName());
-		host_online_button->setWidgetBack("back_button");
-		host_online_button->setWidgetUp("host_lan");
-		host_online_button->setWidgetDown("join");
-		host_online_button->setCallback([](Button&){soundActivate(); createLobby(LobbyType::LobbyOnline);});
-
-		(void)window->addImage(
-		    SDL_Rect{270, 234, 126, 50},
-		    0xffffffff,
-		    "*images/ui/Main Menus/Play/NewGameConnectivity/UI_NewGame_Icon_HostOnline_00B_Unselected.png",
-		    "host_online_image"
-		);
-
-		auto join_button = window->addButton("join");
-		join_button->setSize(SDL_Rect{96, 326, 164, 62});
-		join_button->setBackground("*images/ui/Main Menus/Play/NewGameConnectivity/ButtonStandard/Button_Standard_Default_00.png");
-		join_button->setHighlightColor(makeColor(255, 255, 255, 255));
-		join_button->setColor(makeColor(255, 255, 255, 255));
-		join_button->setText("Lobby Browser");
-		join_button->setFont(smallfont_outline);
-		join_button->setWidgetSearchParent(window->getName());
-		join_button->setWidgetBack("back_button");
-		join_button->setWidgetUp("host_online");
-		join_button->setCallback(createLobbyBrowser);
-
-		(void)window->addImage(
-		    SDL_Rect{270, 324, 120, 68},
-		    0xffffffff,
-		    "*images/ui/Main Menus/Play/NewGameConnectivity/UI_NewGame_Icon_LobbyBrowser_00B_Unselected.png",
-		    "join_image"
-		);
-
-		auto tooltip = window->addField("tooltip", 1024);
-		tooltip->setSize(SDL_Rect{106, 398, 300, 48});
-		tooltip->setFont(smallfont_no_outline);
-		tooltip->setColor(makeColor(91, 76, 50, 255));
-		tooltip->setJustify(Field::justify_t::CENTER);
-		tooltip->setText("Help text goes here.");
+	    if (skipintro) {
```

Comment:
There's a bool FirstTimePrompt.showFirstTimePrompt for this, which gets written to disk in the tutorial_scores.json "first_time_prompt" with the Tutorial.writeToDocument() on button click

Recommend adding it instead of skipintro - could close the game prematurely at title screen (couldn't press 'A' or something on controller) and skipintro would be false next boot?

Reply:
Fixed